samples/daily-prebuilt/sticky-positioning/index.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Daily Prebuilt - Sticky positioning example</title>
+    <script src="https://unpkg.com/@daily-co/daily-js"></script>
+    <link rel="stylesheet" href="style.css" />
+    <script src="index.js"></script>
+  </head>
+  <body>
+    <main>
+      <h1>Daily Prebuilt - Sticky positioning example</h1>
+
+      <h2>Enter a Daily room URL to join a Daily call</h2>
+      <form id="joinForm">
+        <label for="url">Room URL</label>
+        <input id="url" type="text" />
```

Comment:
I know we don't really want to do any validation here for simplicity, but could we just mark this field as required and make the type "url" to at least have the default checks that the URL is filled in and is at least a URL?

Reply:
(Actually I cannot remember if I also mentioned this in the outline and you had a reason for not including that kind of validation here... sorry if I did and just forgot!)

Reply:
No reason other than to keep this supppper basic, but I can definitely update the input with those changes!

Reply:
Let's go for it! I think a couple of extra HTML attributes won't complicate it much and they'll provide significant benefit

Reply:
Updated 🚀